helpers/Cargo.toml

```diff
@@ -14,5 +14,4 @@ milli = { path = "../milli" }
 stderrlog = "0.5.1"
 structopt = { version = "0.3.26", default-features = false }
 
-[target.'cfg(target_os = "linux")'.dependencies]
-jemallocator = "0.3.2"
+mimalloc = { version = "0.1.29", default-features = false }
```

Comment:
Could you add the newline here, please?
FYI there is a setting for that in sublime text:
```json
{ "ensure_newline_at_eof_on_save": true }
```

Reply:
yes, thanks for the setting! it's set now 😄